### DISCUSSION TITLE
Can I change the user tag name on the remote dialog box

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#3015] [Ideas]
During the remote process, if you need to modify the name, you must go back to the client to modify it. It is a little inconvenient. It is very good to modify the name directly by right-clicking on the remote dialog box
![Uploading QQ图片20230131110918.png…]()

**Comment by rustdesk** [comment]
PR is welcome

**Comment by youhers** [comment]
I am sorry that I do not have any basis and look forward to this modification later
